plugins/acss-kit/scripts/verify_html_integration.py

```
@@ -0,0 +1,447 @@
+        if any(part in SKIP_DIRS for part in p.relative_to(root).parts):
+            continue
+        if p.suffix.lower() in PAGE_EXTENSIONS:
+            yield p
+
+
+def read_target(root: Path) -> dict:
+    target = root / CONFIG_FILENAME
+    if target.is_file():
+        try:
+            return json.loads(target.read_text(encoding="utf-8"))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)